lld/test/wasm/mutable-global-exports.s

```diff
@@ -101,6 +101,9 @@ _start:
 # CHECK-ALL-NEXT:      - Name:            __table_base
 # CHECK-ALL-NEXT:        Kind:            GLOBAL
 # CHECK-ALL-NEXT:        Index:           10
+# CHECK-ALL-NEXT:      - Name:            __wasm_first_page_end
+# CHECK-ALL-NEXT:        Kind:            GLOBAL
+# CHECK-ALL-NEXT:        Index:           11
```

Comment:
This makes me wonder about `--export-all` .. is it even needed?  Should it maybe not apply to these internal symbols?  Unrelated to this change though.